vignettes/package_extensions.Rmd

```diff
@@ -54,15 +22,16 @@ admiralci_homepage <- "https://pharmaverse.github.io/admiralci"
 
 1. From the companies that agree to co-develop, identify a lead from each. One company should act as the driver for the overall package extension and put forward a product owner and technical lead who ultimately have final say on any contentious decisions. The product owner would cover project decisions (e.g. around scope and priorities), whereas the technical lead would cover technical decisions (e.g. around design and implementation). _The technical lead should either: a) already be significantly involved in the `{admiral}` core development team as a developer/contributor, or b) join the core development team simultaneously._ This is so as to ensure that the design is kept true to the manifesto and of consistent style and quality with respect to `{admiral}` standards.
 1. Agree on a charter and expectations of each company, e.g. we usually ask for at least 3 developers with at least 25% capacity and a mix of R, GitHub and TA experience. Within the charter make sure the scope and timelines are clear. _It is important here not to try to boil the ocean. Focus first on the very common endpoints required as a foundation and then the package can build up from here via contributions from both the co-development companies and also the wider across-industry admiral community. If useful, the `{admiralonco}` charter could be shared as a guide._
-1. Each company should start to identify the required developer resources. Then each developer is required to complete the `{admiral`} [dummy issue for onboarding](https://github.com/pharmaverse/admiral/issues/1839), as well as reading up on the `{admiral}` `r dyn_link("site", admiral_homepage)` - especially the developer guides, which all need to be followed for package extensions.
+1. Each company should start to identify the required developer resources. Then each developer is required to complete the `{admiral`} [dummy issue for onboarding](https://github.com/pharmaverse/admiral/issues/1839), as well as reading up on the [admiraldev documentation](https://pharmaverse.github.io/admiraldev/main/index.html), - especially the developer guides, which all need to be followed for package extensions.
 1. Optionally it can be useful to host a kick-off meeting to decide how the team will work, for which we recommend agile/scrum practices.
 1. Set up a "admiralxxx_dev" channel on Slack to add all team members to for informal team chat, and agree a way to share working documents across the co-development team. 
+1. Set up a folder on the [pharmaverse admiral MS Teams site](https://teams.microsoft.com/l/team/19%3a5gA-caRueo_CpnVwIlr6EdZVgGkNcn0EIBFD8l5Fz3Q1%40thread.tacv2/conversations?groupId=4bae92df-b2ca-4069-b52a-fe1f8e6244c5&tenantId=2e09e1c5-5c3d-4bca-97c4-573a5695f550) for file sharing - please contact Michael Rimler (michael.s.rimler@gsk.com) for help with this.
```

Comment:
i would delete this in favour of what i added above - the link wouldn't work for people without access anyway, so Michael would get them on to this

Reply:
updated